config/policies.js

```diff
@@ -89,5 +89,6 @@ module.exports.policies = {
 
   TestController: {
     '*': 'developmentOrStagingOnly',
+    'wipeDatabase': 'deleteSession'
```

Comment:
This policy was previously unused, maybe because it was implemented incorrectly. My idea here is that whenever we're wiping the database, we should also clear out the cypress user's session. That should prevent cross-test interactions via carryover session data. If it works, this would resolve #1191

Reply:
Confirmed: resolves #1191